pkg/function/backup_data_using_kopia_server.go

```diff
@@ -254,3 +267,11 @@ func userCredentialsAndServerTLS(tp *param.TemplateParams) (string, string, erro
 	}
 	return string(userCredJSON), string(certJSON), nil
 }
+
+func checkHostnameExistsInUserAccessMap(userAccessMap map[string]string, hostname string) error {
+	// check if hostname is provided in the repository server exists in the user access map
```

Comment:
this is the `hostname is provided` or it exists in the user access map?

Reply:
multiple mapping of hostnames and passwords could be present in User Access Map inside Repository Server User Access Credentials Secret, So the end user has to provide one of the hostname which he/she wants to use for connecting with kopia repository server